no-kendoui-professional-in-package-manager.md

```diff
@@ -0,0 +1,67 @@
+
+---
+title: Kendo UI Professional Is Not Available in the AppBuilder Package Manager
+description: Kendo UI Professional Is Not Available in the AppBuilder Package Manager When Using a DevCraft License
+type: Troubleshooting
+page_title: Kendo UI Professional Is Not Available in the AppBuilder Package Manager
+slug: no-kendoui-professional-in-package-manager
+position: 0
+tags: devcraft, kendo, package manager, manage packages, appbuilder, license
+ticketid: 1017056, 1017115
+
+---
+
+## Environment 
+<table>
+  <tr>
+    <td>Service</td>
+    <td>Code</td>
+  </tr>
+  <tr>
+    <td>Feature</td>
+    <td>Package Manager</td>
+  </tr>
+  <tr>
+    <td>Mobile development type</td>
+    <td>Hybrid (Cordova app)</td>
+  </tr>
+  <tr>
+    <td>Code (AppBuilder) client</td>
+    <td>All</td>
+  </tr>
+</table>
+## Description
+When having two subscriptions, the Devcraft subscription gives you the possibility to use the Kendo UI Professional. However, the Kendo UI Professional package cannot be installed in an AppBuilder app using the Package Manager.
+
+The Manage Packages window lets you install the Kendo UI Core package only. The Kendo UI Professional package is disabled, even though you have a valid Kendo UI Professional subscription.
+
+## Cause
+Currently, there is a limitation in the Package Manager that prevents it from recognizing a Kendo UI license from a bundle different than Telerik Platform. In Telerik Platform, only the Business and Enterprise subscriptions include a Kendo UI Professional license. So, if you have Telerik Platform Professional or Developer subscription in combination with a DevCraft subscription, the Package Manager won't let you install the Kendo UI Professional bundle.
+
+## Solution
+There isn't a way to automatically add the Kendo UI Professional package in your app in this scenario. However, you can do this manually, as described in the next section.
+
+## Workaround
+You can manually add the files to your project and reference them. 
+
+To do this:
+
+1. Go to Your Account at <a href="http://www.telerik.com" target="_blank" title="Telerik">http://www.telerik.com</a>.
+2. Click **Products & Subscriptions** -> **DevCraft Complete/Ultimate**.
+3. Click **Download Installer and other resources**.
+4. Find **Kendo UI Professional** in the list and click **Browse all product files**.
+5. Download the *telerik.kendoui.professional....* zip file.
```

Comment:
If the periods stand for a version, it's better to provide the version format using XXXXs, or to skip it altogether and go with something like `telerik.kendoui.professional.*.zip`